glean-core/ios/GleanTests/TestUtils.swift

```diff
@@ -41,3 +41,19 @@ func stubServerReceive(callback: @escaping (String, [String: Any]?) -> Void) {
         )
     }
 }
+
+/// Stringify a JSON object and if unable to, just return an empty string.
+func JSONStringify(_ json: Any) -> String {
```

Comment:
Added this debugging purposes, but left because I think it might be helpful for future debugging too.